### ISSUE TITLE
Support for livenessProbe or health check in podman kube play

### ISSUE THREAD

**Issue by gcs278** [containers/podman#8928]
**Is this a BUG REPORT or FEATURE REQUEST? (leave only one on its own line)**

/kind feature

**Description**

Currently, there isn't a way to do `--health-cmd` with a pod that was started by `podman kube play`. I have a need to use a health check with `podman kube play` command, but I can't find any way to accomplish this. My assumption is that `--health-cmd` can somehow be translate to Kubernetes's livenessProbes and vice versa.

Along with this - I was going to suggest preserving the Dockerfile HEALTHCHECK, but it does appear that Kubernetes is explicitly disabling use of Dockerfile healthchecks, so I can understand not implementing that.

Also as a secondary comment, I've struggled to understand what podman play kube support due to the lack of documentation. I figure it's a "go to kubernetes docs" or do a `podman generate kube`; however, when something that `podman run` and/or kubernetes supports, but `podman play kube` doesn't, I turn to searching the source code. Am I wrong or is there no docs on `podman play kube`? Would it be smart to warn users if you try to use a function Kubernetes that is unsupported in `podman play kube`?

Thank you for the support.

**Steps to reproduce the issue:**

man podman-play-kube
No support for livenessProbe or --health-cmd

**Describe the results you received:**

N/A

**Describe the results you expected:**

N/A

**Additional information you deem important (e.g. issue happens only occasionally):**

**Output of `podman version`:**

```
Version:      2.2.1
API Version:  2.1.0
Go Version:   go1.14.7
Built:        Thu Dec 10 13:26:48 2020
OS/Arch:      linux/amd64
```

**Output of `podman info --debug`:**

```
host:
  arch: amd64
  buildahVersion: 1.18.0
  cgroupManager: systemd
  cgroupVersion: v1
  conmon:
    package: conmon-2.0.21-1.el8.x86_64
    path: /usr/bin/conmon
    version: 'conmon version 2.0.21, commit: f619ab8ef5f69bd40bb75ed64f3e1dace1815c22-dirty'
  cpus: 10
  distribution:
    distribution: '"rhel"'
    version: "8.2"
  eventLogger: journald
  hostname: rhel82
  idMappings:
    gidmap: null
    uidmap: null
  kernel: 4.18.0-193.el8.x86_64
  linkmode: dynamic
  memFree: 150331392
  memTotal: 3884625920
  ociRuntime:
    name: crun
    package: crun-0.16-1.el8.x86_64
    path: /usr/bin/crun
    version: |-
      crun version 0.16
      commit: eb0145e5ad4d8207e84a327248af76663d4e50dd
      spec: 1.0.0
      +SYSTEMD +SELINUX +APPARMOR +CAP +SECCOMP +EBPF +YAJL
  os: linux
  remoteSocket:
    path: /run/podman/podman.sock
  rootless: false
  slirp4netns:
    executable: ""
    package: ""
    version: ""
  swapFree: 4185374720
  swapTotal: 4190105600
  uptime: 1494h 45m 43.14s (Approximately 62.25 days)
registries:
  search:
  - registry.access.redhat.com
  - registry.redhat.io
  - docker.io
store:
  configFile: /etc/containers/storage.conf
  containerStore:
    number: 56
    paused: 0
    running: 2
    stopped: 54
  graphDriverName: overlay
  graphOptions: {}
  graphRoot: /var/lib/containers/storage
  graphStatus:
    Backing Filesystem: xfs
    Native Overlay Diff: "true"
    Supports d_type: "true"
    Using metacopy: "false"
  imageStore:
    number: 44
  runRoot: /var/run/containers/storage
  volumePath: /var/lib/containers/storage/volumes
version:
  APIVersion: 2.1.0
  Built: 1607624808
  BuiltTime: Thu Dec 10 13:26:48 2020
  GitCommit: ""
  GoVersion: go1.14.7
  OsArch: linux/amd64
  Version: 2.2.1
```

**Package info (e.g. output of `rpm -q podman` or `apt list podman`):**

```
podman-2.2.1-1.el8.x86_64
```

**Have you tested with the latest version of Podman and have you checked the Podman Troubleshooting Guide?**

Yes

**Additional environment details (AWS, VirtualBox, physical, etc.):**

Libvirt/KVM

**Comment by rhatdan** [comment]
This seems like a good request.  Currently the core team is concentrating on Bug fixes and stabilization of podman 3.0.
Opening a PR is always the best way to get new features in, if you are interested.  I will add the help wanted flag to see if someone in community would like to work on this.

**Comment by zhangguanzhang** [comment]
ref: https://github.com/containers/podman/pull/10956 had done for this
/close
